docker-compose.yml

```diff
@@ -343,6 +384,7 @@ services:
     command: sh -c ". /tmp/dev-tools/startup/startup.sh && scripts/start.sh"
     env_file: .env # used by the startup script
     environment:
+      audius_db_url: "postgres://postgres:postgres@discovery-provider-db:5432/postgres"
```

Comment:
So it seems the discovery postgres actually runs inside the discovery container.

So we should remove this setting.  I think the comms could be pointed at a discovery container instead.